system-tests/test/browser_crash_handling_spec.js

```diff
@@ -0,0 +1,49 @@
+const systemTests = require('../lib/system-tests').default
+
+describe('Browser Crash Handling', () => {
+  systemTests.setup({
+    settings: {
+      e2e: {},
+    },
+  })
+
+  // It should fail the chrome_tab_crash spec, but the simple spec should run and succeed
+  context('when the tab crashes in chrome', () => {
+    systemTests.it('fails', {
+      browser: 'chrome',
```

Comment:
should we add one for edge too? Guessing no since we treat chrome/edge the same?

Reply:
Yeah, edge and chrome behave the same, no harm in testing it though 🤔

Reply:
Apparently edge isn't available in system tests? 🤷‍♂️

Reply:
@mjhenkes Its likely because we don't run test on a window instance in CI